buildkite/bazelci.py

```diff
@@ -2290,6 +2294,19 @@ def set_env_var(config_key, env_var_name):
     print_pipeline_steps(pipeline_steps, handle_emergencies=not is_downstream_project)
 
 
+def show_gerrit_review_link(pipeline_steps):
+    text = "The transformed code used in this pipeline can be found under https://bazel-review.googlesource.com/q/%s" \
```

Comment:
This comment is not relevant right now, but the link will break once we add a pipeline that runs code from a Git-on-Borg host other than "bazel". 

I think we can easily make it future-proof by reading the host name from BUILDKITE_REPO, though.

Background: Git-on-Borg has a concept called hosts which is pretty much equivalent to GitHub orgs. Each host can contain multiple repositories. Each host has two domains, https://HOST.googlesource.com/ and 
https://HOST-review.googlesource.com/.

Currently we only have a single host called "bazel" with several repositories ("bazel", "rules_typescript" etc), but we might have to read from other hosts in the future.

Reply:
Ok, this comment is actually already relevant since this pipeline reads from the "gerrit" host: https://buildkite.com/bazel/gerrit/builds/5856#61e074fe-b176-446d-abee-f091498c13b1

Reply:
From the "Environment" tab: BUILDKITE_REPO="https://gerrit.googlesource.com/gerrit.git"